jobs/e2e_node/containerd/containerd-release-1.5/image-config.yaml

```diff
@@ -0,0 +1,9 @@
+images:
+  ubuntu:
+    image: ubuntu-gke-2004-1-21-v20210722 # docker 19.03.8 / containerd 1.4.3
```

Comment:
Why is this image using containerd 1.4.3?

Reply:
This is the version included in the image by default. The job installs containerd 1.5 before running tests.

Reply:
Just for my understanding; how is ctrd1.5 installed?

Reply:
If you go to the job configuration ([here](https://github.com/kubernetes/test-infra/pull/23215/files#diff-d5b95dd765e75cb94ffa934ececa20b40aecb9872c6dd284efa3cb5db903ff36R78)), you'll notice that it downloads containerd from release 1.5 and then installs it.

Reply:
thank you for explaining! Adding comments will be useful, but it is upto you.